File: ransor/Ransor.java
// Ransor.java
// Jungle Monster - Oblivion

public class Ransor {
    private String name = "Ransor";
    private int health = 10000;
    private int armor = 300;
    private int damage = 800;

    public String attack() {
        return "Ransor strikes with demonic fury!";
    }

    public String grantBuff(String championClass) {
        switch (championClass) {
            case "Slayer": return "Void Essence";
            case "Juggernaut": return "Stamina";
            case "Bruiser": return "Berzerker";
            default: return "No buff";
        }
    }

    public static void main(String[] args) {
        Ransor r = new Ransor();
        System.out.println(r.name + " attacks: " + r.attack());
        System.out.println("Buff for Slayer: " + r.grantBuff("Slayer"));
    }
}

File: ransor/Ransor.py
"""
Ransor - Jungle Monster (Oblivion)
Provides: Void Essence to Slayers, Stamina to Juggernauts, Berzerker to Bruisers
"""

class Ransor:
    def __init__(self):
        self.name = "Ransor"
        self.health = 10000
        self.armor = 300
        self.damage = 800
        self.buff_given = {
            "Slayer": "Void Essence",
            "Juggernaut": "Stamina",
            "Bruiser": "Berzerker"
        }

    def attack(self):
        return "Ransor strikes with demonic fury!"

    def grant_buff(self, champion_class):
        return self.buff_given.get(champion_class, "No buff")

if __name__ == "__main__":
    r = Ransor()
    print(f"{r.name} attacks: {r.attack()}")
    print("Buff for Slayer:", r.grant_buff("Slayer"))
